pvlib/tests/conftest.py

```diff
@@ -16,25 +17,25 @@
 # decorator takes one argument: the base version for which it should fail
 # for example @fail_on_pvlib_version('0.7') will cause a test to fail
 # on pvlib versions 0.7a, 0.7b, 0.7rc1, etc.
-# test function may not take args, kwargs, or fixtures.
-def fail_on_pvlib_version(version):
+# args and kwargs will be passed to the function being decorated.
+def fail_on_pvlib_version(version, *args, **kwargs):
```

Comment:
`*args, **kwargs` and associated comment not needed here anymore

Reply:
Yeah, this should be enough: https://github.com/Peque/pvlib-python/commit/44e86c20ab80fb19246058beb78a9c361ab95300

Reply:
I still think the `*args, **kwargs` here should be removed.